receptor_affinity/mesh.py

```diff
@@ -393,7 +384,7 @@ def start(self, wait=True):
             if self.use_diag_node:
                 self.diag_node.add_peer(self.find_controller()[0])
                 self.nodes[self.diag_node.name].connections.append(self.find_controller()[0].name)
-            wait_for(self.validate_all_node_routes, delay=6, num_sec=30)
+            self.settle()
             # for name, node in self.nodes.items():
```

Comment:
With the `settle` method, can these commented lines be dropped?

Reply:
Sure, I expect so. I'll do that. :smile: